src/crossroads/templates/locations.html

```diff
@@ -0,0 +1,34 @@
+{% extends "material/base.html" %}
+{% import "material/wtf.html" as wtf %}
+
+{% block title %}Crossrads!{% endblock %}
+
+{% block content %}
+<div class="container">
+        <div class="row">
+                <div class="col s12 m6">
+                    <div class="card">
+                        <div class="card-content">
+                            <span class="card-title black-text">Crossroads!</span>
+                                <br><br>
+                                <center>
+                                <img src="http://maps.google.com/mapfiles/ms/icons/blue-dot.png" alt="Blue"><p>{{name1}}</p>
```

Comment:
the `name1` and `name2` could be slightly more explanatory since name even with context tell nothing. I suggest rename it to `user_name_one` `user_name_two` since it already describes what content this variables will contain.